pkg/Microsoft.NETCore.Platforms.Future/Microsoft.NETCore.Platforms.Future.pkgproj

```diff
@@ -1,7 +1,6 @@
 ﻿<Project DefaultTargets="Build">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
   <PropertyGroup>
-    <PackageVersion>3.0.0</PackageVersion>
```

Comment:
This shouldn't be removed.  It was removed in 5.0 because we unified repo-wide-package-version with product version.  It's still not unified in 3.1.  Set this to `3.1.0`.